Source/FillableLoader.swift

```diff
@@ -16,26 +16,25 @@ open class FillableLoader: UIView, CAAnimationDelegate {
     internal var animate: Bool = false
     internal var extraHeight: CGFloat = 0
     internal var oldYPoint: CGFloat = 0
-    internal let mainBgColor = UIColor(white: 0.2, alpha: 0.6)
     internal weak var loaderSuperview: UIView?
-
+    
```

Comment:
Please remove all this kind of changes

Enable the following checks
![screen shot 2016-11-08 at 11 22 02](https://cloud.githubusercontent.com/assets/7887319/20095405/ac2589c6-a5a5-11e6-9117-04dea4e7dde1.png)

And then, select all code (CMD + A), and Re-indent (Ctrl + I)